xain_fl/grpc/coordinator.py

```diff
@@ -658,13 +658,26 @@ def monitor_heartbeats(
         time.sleep(next_expiration)
 
 
-def serve() -> None:
+def serve(
```

Comment:
One last thing, can we please change the `serve` function signature in a way that it accepts only a coordinator, a host, and a port ([example here](https://github.com/xainag/xain-fl/blob/hackaton-topic1/xain_fl/grpc/coordinator.py#L754)). It makes it more flexible. For example we could call the serve function with a mocked coordinator for easier testing.

Reply:
Done. Please review again.